Bottom: 2dae1fb79247ac27285f658b6d2f48c20e2b7bd1
Top:    6edb2f683e6f3f27d143bb76b9f10ede08f8c141
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-23 17:50:40 -0500

Refresh of update-gitlab-to-10-1

---
diff --git a/ansible-playbook/gitlab.yml b/ansible-playbook/gitlab.yml
index 03c67ee4..d958a565 100644
--- a/ansible-playbook/gitlab.yml
+++ b/ansible-playbook/gitlab.yml
@@ -23,41 +23,47 @@
     - "{{ gitlab_backup_storage }}"
     - "{{ gitlab_restore_dir }}"
 
-  - name: Gitlab 7.14.3 (Sameersbn 7)
+  # - name: Run bacula-restores symlinks
+  #   include_role:
+  #     name: bacula-restores
+
+  - name: Gitlab Sameersbn Upgrade Path
+    loop:
+    - { last: Null,      version: "7.14.3",  tag: "7.14.3" }
+    - { last: "7.14.3",  version: "8.0.5",   tag: "8.0.5"  }
+    loop_control:
+      loop_var: play_item
     vars:
-      gitlab_version: "7.14.3"
-      docker_image_tag: "7.14.3"
-      docker_name_prefix: ''
-      upgrade: false
-      restore: true
-    include_role: 
+      old_prefix: 'sameersbn{{play_item.last}}-'
+      docker_name_prefix: 'sameersbn{{play_item.version}}-'
+      gitlab_version: '{{play_item.version}}'
+      gitlab_image_tag: '{{play_item.tag}}'
+      upgrade: '{{play_item.last is not none}}'
+    include_role:
       name: docker-gitlab
   
   - name: Gitlab (Sameersbn 8)
     vars:
       gitlab_version: "8.0.5"
-      gitlab_image_tag: "8.0.5"
-      docker_name_prefix: 'sameersbn8-'
-      upgrade: true
-      old_prefix: ''
+      docker_name_prefix: 'sameersbn8.0.5-'
     block:
-    - name: Run Gitlab
-      include_role:
-        name: docker-gitlab
     - name: Backup Gitlab
       command: >
         docker exec -u git {{docker_name_prefix}}gitlab bundle exec rake gitlab:backup:create
+
     - name: Find backup files
       find:
         path: '{{gitlab_backup_dir}}'
         pattern: '*_gitlab_backup.tar'
       register: backups
+
     - name: Move backups to storage, normalize name
       loop: '{{ backups.files }}'
       vars:
         backup_file: '{{ item.path | basename | replace("gitlab_backup.tar", gitlab_version ~ "_gitlab_backup.tar") }}'
       command: >
         mv "{{ item.path }}" "{{gitlab_backup_storage}}/{{ backup_file }}"
+
     - name: Shutdown Gitlab
       docker_container: 
         name: '{{ docker_name_prefix }}{{ item }}'
@@ -67,69 +73,24 @@
       - gitlab-redis
       - gitlab-db
 
-  - name: Gitlab (Omnibus 8)
-    vars:
-      docker_name_prefix: 'omnibus8.0-'
-      gitlab_version: "8.0.5"
-      gitlab_image_tag: "8.0.5-ce.0"
-      upgrade: false
-      restore: true
-    block:
-    - name: Find relevant backups
-      find:
-        path: "{{gitlab_backup_storage}}"
-        pattern: "*_{{gitlab_version}}_*"
-      register: backups
-    - file:
-        src: '{{ item.path }}'
-        dest: '{{ item.path | replace(gitlab_backup_storage,gitlab_restore_dir) }}'
-        state: link
-        force: yes
-        mode: '{{ item.mode }}'
-      loop: '{{ backups.files }}'
-    - include_role:
-        name: gitlab-omnibus
-
-  - name: Gitlab Omnibus Upgrade Path
-    loop:
-    - { last: "8.0.5",  version: "8.17.8",  tag: "8.17.8-ce.0"  } # Health-check failure
-    - { last: "8.17.8", version: "9.5.10",   tag: "9.5.10-ce.0"  }
-    - { last: "9.5.10", version: "10.8.4",  tag: "10.8.4-ce.0"  }
+  # - name: Run bacula-restores symlinks
+  #   include_role:
+  #     name: bacula-restores
 
-    # - { last: "8.0.5", version: "8.1",   tag: "8.1.4-ce.0"   }
-    # - { last: "8.1",   version: "8.2",   tag: "8.2.5-ce.2"   }
-    # - { last: "8.2",   version: "8.3",   tag: "8.3.10-ce.2"  }
-    # - { last: "8.3",   version: "8.4",   tag: "8.4.11-ce.2"  }
-    # - { last: "8.4",   version: "8.5",   tag: "8.5.13-ce.3"  } # reconfigure fail after uri
-    # - { last: "8.5",   version: "8.6",   tag: "8.6.9-ce.3"   }
-    # - { last: "8.6",   version: "8.7",   tag: "8.7.9-ce.1"   }
-    # - { last: "8.7",   version: "8.8",   tag: "8.8.9-ce.0"   } # Needs additional restart
-    # - { last: "8.8",   version: "8.9",   tag: "8.9.11-ce.0"  } # Needs additional restart
-    # - { last: "8.9",   version: "8.10",  tag: "8.10.13-ce.0" }
-    # - { last: "8.10",  version: "8.11",  tag: "8.11.11-ce.0" }
-    # - { last: "8.11",  version: "8.12",  tag: "8.12.13-ce.0" }
-    # - { last: "8.12",  version: "8.13",  tag: "8.13.12-ce.0" }
-    # - { last: "8.13",  version: "8.14",  tag: "8.14.10-ce.0" }
-    # - { last: "8.14",  version: "8.15",  tag: "8.15.8-ce.0"  }
-    # - { last: "8.15",  version: "8.16",  tag: "8.16.9-ce.0"  } # Health-check failure
-    # - { last: "8.16",  version: "8.17",  tag: "8.17.8-ce.0"  } # Health-check failure
-    # - { last: "8.17",  version: "9.0",   tag: "9.0.13-ce.0"  } # Health-check failure
-    # - { last: "9.0",   version: "9.1",   tag: "9.1.10-ce.0"  } # docker healthcheck introduced
-    # - { last: "9.1",   version: "9.2",   tag: "9.2.10-ce.0"  }
-    # - { last: "9.2",   version: "9.3",   tag: "9.3.11-ce.0"  }
-    # - { last: "9.3",   version: "9.4",   tag: "9.4.7-ce.0"   }
-    # - { last: "9.4",   version: "9.5",   tag: "9.5.10-ce.0"  }
-    # - { last: "9.5",   version: "10.0",  tag: "10.0.7-ce.0"  }
-    # - { last: "10.0",  version: "10.1",  tag: "10.1.7-ce.0"  }
-    # - { last: "10.1",  version: "10.2",  tag: "10.2.8-ce.0"  }
-    # - { last: "10.8",  version: "11.0",  tag: "11.0.0-rc13.ce.0"  }
-    loop_control:
-      loop_var: play_item
-    vars:
-      old_prefix: 'omnibus{{play_item.last}}-'
-      docker_name_prefix: 'omnibus{{play_item.version}}-'
-      gitlab_version: '{{play_item.version}}'
-      gitlab_image_tag: '{{play_item.tag}}'
-      upgrade: true
-    include_role:
-      name: gitlab-omnibus
+  # - name: Gitlab Omnibus Upgrade Path
+  #   loop:
+  #   - { last: Null,     version: "8.0.5",  tag: "8.0.5-ce.0"   }
+  #   - { last: "8.0.5",  version: "8.17.8", tag: "8.17.8-ce.0"  }
+  #   - { last: "8.17.8", version: "9.0.13", tag: "9.0.13-ce.0"  }
+  #   - { last: "9.0.13", version: "9.5.10", tag: "9.5.10-ce.0"  }
+  #   - { last: "9.5.10", version: "10.8.4", tag: "10.8.4-ce.0"  }
+  #   loop_control:
+  #     loop_var: play_item
+  #   vars:
+  #     old_prefix: 'omnibus{{play_item.last}}-'
+  #     docker_name_prefix: 'omnibus{{play_item.version}}-'
+  #     gitlab_version: '{{play_item.version}}'
+  #     gitlab_image_tag: '{{play_item.tag}}'
+  #     upgrade: '{{play_item.last is not none}}'
+  #   include_role:
+  #     name: gitlab-omnibus
diff --git a/ansible-playbook/group_vars/docker-hosts/docker.yml b/ansible-playbook/group_vars/docker-hosts/docker.yml
index f9153cdc..4623feba 100644
--- a/ansible-playbook/group_vars/docker-hosts/docker.yml
+++ b/ansible-playbook/group_vars/docker-hosts/docker.yml
@@ -17,7 +17,7 @@ docker_restore_config_base_dir: '/etc/docker_container.conf/{{ docker_image_tag
 docker_backup_dir: '/tmp/docker'
 
 # docker service restart policy
-docker_restart_policy: 'no'
+docker_restart_policy: 'always'
 
 # Network names
 docker_network_frontend: 'frontend'
diff --git a/ansible-playbook/host_vars/testdaedalus.novatech-llc.com/docker.yml b/ansible-playbook/host_vars/testdaedalus.novatech-llc.com/docker.yml
index df7bb7de..30d74e22 100644
--- a/ansible-playbook/host_vars/testdaedalus.novatech-llc.com/docker.yml
+++ b/ansible-playbook/host_vars/testdaedalus.novatech-llc.com/docker.yml
@@ -1,3 +1,5 @@
 ---
 
 docker_storage_driver: devicemapper
+
+docker_restart_policy: 'no'
diff --git a/docker/docker-gitlab-data/Dockerfile b/docker/docker-gitlab-data/Dockerfile
index 2571036e..43edab83 100644
--- a/docker/docker-gitlab-data/Dockerfile
+++ b/docker/docker-gitlab-data/Dockerfile
@@ -3,8 +3,7 @@
 #
 # Version 0.1
 
-ARG  GITLAB_VERSION=7.14.3
-FROM sameersbn/gitlab:$GITLAB_VERSION
+FROM sameersbn/gitlab:7.14.3
 MAINTAINER Joseph Lutz <Joseph.Lutz@novatechweb.com>
 
 # copy wrapper script
diff --git a/roles/bacula-restores/tasks/main.yml b/roles/bacula-restores/tasks/main.yml
index 6939a2fc..80332789 100644
--- a/roles/bacula-restores/tasks/main.yml
+++ b/roles/bacula-restores/tasks/main.yml
@@ -1,26 +1,22 @@
 ---
 # tasks file for bacula-restores
 
-- name: List bacula storage
-  find:
-    paths: "{{bacula_storage}}"
-    file_type: file
-    recurse: yes
-  register: bacula_restores_list
+# - name: List bacula storage
+#   find:
+#     paths: "{{bacula_storage}}"
+#     file_type: file
+#     recurse: yes
+#   register: bacula_storage_list
 
-- name: Create bacula storage symlinks
-  file:
-    path: '{{ item.path | replace("{{bacula_storage}}","{{bacula_dest}}") | dirname }}'
-    state: directory
-    force: yes
-    mode: 0777
-  with_items: '{{ bacula_restores_list.files }}'
+# - name: Prepare backup file
+#   include_tasks: prepare_backup_file.yml
+#   loop: '{{bacula_storage_list.files}}'
 
-- name: Create bacula storage directories
-  file:
-    src: '{{ item.path }}'
-    dest: '{{ item.path | replace("{{bacula_storage}}","{{bacula_dest}}") }}'
-    state: link
-    force: yes
-    mode: '{{ item.mode }}'
-  with_items: '{{ bacula_restores_list.files }}'
+- name: synchronize backup storage
+  synchronize:
+    src: "{{bacula_storage}}/"
+    dest: "{{bacula_dest}}/"
+    link_dest: "{{bacula_storage}}/"
+    archive: yes
+    delete: yes
+  delegate_to: "{{ inventory_hostname }}"
diff --git a/roles/docker-gitlab/tasks/main.yml b/roles/docker-gitlab/tasks/main.yml
index fe09cd48..d2ceb73c 100644
--- a/roles/docker-gitlab/tasks/main.yml
+++ b/roles/docker-gitlab/tasks/main.yml
@@ -93,6 +93,8 @@
     tag: '{{ docker_image_tag }}'
     path: '{{ docker_projects_dir }}/docker-gitlab'
     force: "{{ docker_image_force_build }}"
+    buildargs:
+      GITLAB_VERSION: '{{gitlab_version}}'
 
 # *****************************************************************************
 # Create the data volumes
diff --git a/roles/gitlab-omnibus/tasks/main.yml b/roles/gitlab-omnibus/tasks/main.yml
index db6cb7bd..6d915a22 100644
--- a/roles/gitlab-omnibus/tasks/main.yml
+++ b/roles/gitlab-omnibus/tasks/main.yml
@@ -121,7 +121,7 @@
   retries: 10
   delay: 30
   until: health is success
-  when: gitlab_version is version('9.1', '<')
+  when: gitlab_version is version('8.17', '<')
 
 - name: Wait for a container to become healthy
   docker_health:
@@ -130,7 +130,7 @@
   retries: 10
   delay: 30
   until: health.Health.Status == "healthy"
-  when: gitlab_version is version('9.1', '>=')
+  when: gitlab_version is version('8.17', '>=')
 
 # *****************************************************************************
 # restore?
@@ -164,3 +164,9 @@
   retries: 10
   delay: 30
   until: check is success
+
+- name: Ensure merge request commits are fetched
+  include_tasks: fetch_merge_requests.yml
+  when: 
+  - gitlab_version is version('9.0', '>=')
+  - gitlab_version is version('10.2', '<')
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-10.0.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-10.0.j2
deleted file mode 100644
index 6288d08c..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-10.0.j2
+++ /dev/null
@@ -1,63 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    encryption: 'plain' # "start_tls" or "simple_tls" or "plain"
-    verify_certificates: true
-    ca_file: ''
-    ssl_version: ''
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-10.1.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-10.1.j2
deleted file mode 100644
index 6288d08c..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-10.1.j2
+++ /dev/null
@@ -1,63 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    encryption: 'plain' # "start_tls" or "simple_tls" or "plain"
-    verify_certificates: true
-    ca_file: ''
-    ssl_version: ''
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-10.2.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-10.2.j2
deleted file mode 100644
index 6288d08c..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-10.2.j2
+++ /dev/null
@@ -1,63 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    encryption: 'plain' # "start_tls" or "simple_tls" or "plain"
-    verify_certificates: true
-    ca_file: ''
-    ssl_version: ''
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.1.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.1.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.1.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.10.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.10.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.10.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.11.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.11.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.11.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.12.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.12.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.12.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.13.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.13.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.13.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.14.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.14.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.14.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.15.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.15.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.15.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.16.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.16.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.16.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.2.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.2.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.2.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.3.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.3.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.3.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.4.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.4.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.4.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.5.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.5.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.5.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.6.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.6.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.6.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.7.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.7.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.7.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.8.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.8.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.8.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-8.9.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-8.9.j2
deleted file mode 100644
index 53c7b06f..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-8.9.j2
+++ /dev/null
@@ -1,60 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    method: 'plain' # "tls" or "ssl" or "plain"
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-9.0.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-9.0.j2
deleted file mode 100644
index 6288d08c..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-9.0.j2
+++ /dev/null
@@ -1,63 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    encryption: 'plain' # "start_tls" or "simple_tls" or "plain"
-    verify_certificates: true
-    ca_file: ''
-    ssl_version: ''
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-9.1.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-9.1.j2
deleted file mode 100644
index 6288d08c..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-9.1.j2
+++ /dev/null
@@ -1,63 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    encryption: 'plain' # "start_tls" or "simple_tls" or "plain"
-    verify_certificates: true
-    ca_file: ''
-    ssl_version: ''
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-9.2.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-9.2.j2
deleted file mode 100644
index 6288d08c..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-9.2.j2
+++ /dev/null
@@ -1,63 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    encryption: 'plain' # "start_tls" or "simple_tls" or "plain"
-    verify_certificates: true
-    ca_file: ''
-    ssl_version: ''
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-9.3.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-9.3.j2
deleted file mode 100644
index 6288d08c..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-9.3.j2
+++ /dev/null
@@ -1,63 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    encryption: 'plain' # "start_tls" or "simple_tls" or "plain"
-    verify_certificates: true
-    ca_file: ''
-    ssl_version: ''
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/roles/gitlab-omnibus/templates/gitlab.rb-9.4.j2 b/roles/gitlab-omnibus/templates/gitlab.rb-9.4.j2
deleted file mode 100644
index 6288d08c..00000000
--- a/roles/gitlab-omnibus/templates/gitlab.rb-9.4.j2
+++ /dev/null
@@ -1,63 +0,0 @@
-external_url 'https://{{ gitlab_hostname }}'
-gitlab_rails['gitlab_ssh_host'] = '{{ gitlab_hostname }}'
-gitlab_rails['time_zone'] = 'America/Chicago'
-gitlab_rails['gitlab_email_enabled'] = true
-gitlab_rails['gitlab_email_from'] = 'gitlab@novatech-llc.com'
-gitlab_rails['gitlab_email_display_name'] = 'GitLab'
-gitlab_rails['gitlab_email_reply_to'] = 'noreply@novatech-llc.com'
-gitlab_rails['gitlab_default_can_create_group'] = true
-gitlab_rails['gitlab_username_changing_enabled'] = true
-gitlab_rails['gitlab_default_projects_features_issues'] = true
-gitlab_rails['gitlab_default_projects_features_merge_requests'] = true
-gitlab_rails['gitlab_default_projects_features_wiki'] = false
-gitlab_rails['gitlab_default_projects_features_snippets'] = true
-gitlab_rails['gravatar_plain_url'] = 'http://cdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-gitlab_rails['gravatar_ssl_url'] = 'https://seccdn.libravatar.org/avatar/%{hash}?s=%{size}&d=identicon'
-
-gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = YAML.load <<-'EOS' # remember to close this block with 'EOS' below
-  main: # 'main' is the GitLab 'provider ID' of this LDAP server
-    label: 'NovaTech'
-    host: '{{ openldap_hostname }}'
-    port: 389
-    uid: 'uid'
-    bind_dn: 'cn=proxyagent,dc=novatech'
-    password: '{{ gitlab_email_password }}'
-    encryption: 'plain' # "start_tls" or "simple_tls" or "plain"
-    verify_certificates: true
-    ca_file: ''
-    ssl_version: ''
-    active_directory: false
-    allow_username_or_email_login: true
-    block_auto_created_users: false
-    base: 'ou=user,dc=novatech'
-    user_filter: '(employeeType=gitlab)'
-    attributes:
-      username: ['uid', 'userid', 'sAMAccountName']
-      email:    ['mail', 'email', 'userPrincipalName']
-      name:       'cn'
-      first_name: 'givenName'
-      last_name:  'sn'
-EOS
-
-gitlab_rails['backup_path'] = "/mnt/backups"
-gitlab_rails['backup_archive_permissions'] = 0644
-gitlab_rails['backup_pg_schema'] = 'public'
-gitlab_rails['backup_keep_time'] = 604800
-
-gitlab_rails['smtp_enable'] = true
-gitlab_rails['smtp_address'] = "{{ exim4_hostname }}"
-gitlab_rails['smtp_port'] = 465
-gitlab_rails['smtp_user_name'] = "gitlab"
-gitlab_rails['smtp_password'] = "{{ gitlab_email_password }}"
-gitlab_rails['smtp_domain'] = "novatech-llc.com"
-gitlab_rails['smtp_authentication'] = "login"
-gitlab_rails['smtp_enable_starttls_auto'] = true
-gitlab_rails['smtp_tls'] = true
-
-nginx['enable'] = true
-nginx['redirect_http_to_https'] = true
-nginx['ssl_certificate'] = "/etc/ssl/private/nginx.crt"
-nginx['ssl_certificate_key'] = "/etc/ssl/private/nginx.key"
-nginx['ssl_dhparam'] = "/etc/ssl/private/dhparam.pem" # Path to dhparams.pem, eg. /etc/ssl/private/dhparams.pem
-nginx['listen_port'] = 443
diff --git a/vmdaedalus/.gitignore b/vmdaedalus/.gitignore
index c148b6f5..b3d3503d 100644
--- a/vmdaedalus/.gitignore
+++ b/vmdaedalus/.gitignore
@@ -1,2 +1 @@
-bacula-restores/
 vmdaedalus-virtualbox.box
